tools/install_bazel.sh

```diff
@@ -21,6 +20,6 @@ else
     OS=$(uname -s | tr '[:upper:]' '[:lower:]')
     ARCH=$(uname -m) && [ "$ARCH" != "aarch64" ] || ARCH="arm64"
     echo "Downloading bazel-${BAZEL_VERSION}-${OS}-${ARCH} to ${BAZEL}"
-    ${SUDO} curl -sfL "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-${OS}-${ARCH}" -o ${BAZEL}
-    ${SUDO} chmod +x ${BAZEL}
+    ${SUDO} curl -sfL "https://github.com/bazelbuild/bazelisk/releases/download/${BAZEL_VERSION}/bazel-${OS}-${ARCH}" -o "${BAZEL}"
+    ${SUDO} chmod +x "${BAZEL}"
```

Comment:
Are we actually running this (vs. `tools/bazelisk.sh`?)

Reply:
renamed the script :heavy_check_mark: